sympy/logic/tests/test_boolalg.py

```diff
@@ -770,3 +770,12 @@ def test_issue_8571():
                 raises(TypeError, lambda: o/t)
                 raises(TypeError, lambda: o**t)
                 o, t = t, o  # do again in reversed order
+
+
+def test_expand_relational():
+    n = symbols('n', negative=True)
+    p, q = symbols('p q', positive=True)
+    r = ((n + q*(-n/q + 1))/(q*(-n/q + 1)) < 0)
+    assert r is not S.false
```

Comment:
```
>>> r = x < 0
>>> r is not S.false
True
>>> r is S.true
False
```

(They are only the same if the relational evaluates. I am affirming here that the relational hasn't evaluated.)